### PR TITLE
ENH: Added ReadAnImage Python script

### PR DESCRIPTION
**Pull request by kian-weimer**
Created a Python ReadAnImage example based off of the existing C++ example.

### CODE REVIEW

**Comment by kian-weimer** [comment]
Would it be preferable to have multiple examples in one PR?

**Review by dzenanz — APPROVED** [review]
I am fine with one PR per example. Others might have a different preference, but it does not matter much. If you do that, please have each new example in a commit of its own.

**Review by dzenanz — APPROVED** [review]

**Comment by dzenanz** [comment]
Could you rebase on master? The log should be shorter then.

**Comment by kian-weimer** [comment]
Rebased. Thanks for the tip. I'm not sure what the issue is yet.

File: src/IO/ImageBase/ReadAnImage/Code.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```
Comment (thewtex) [inline src/IO/ImageBase/ReadAnImage/Code.py:1]: The license header is missing

**Review by thewtex — DISMISSED** [review]
Please copy the license header from another Python example.

File: src/IO/ImageBase/ReadAnImage/Code.py (hunk 1 above)
Comment (kian-weimer) [reply src/IO/ImageBase/ReadAnImage/Code.py:1]: Sorry I missed this! 
What's the preferred method for fixing this on my other already committed PRs?

Comment (dzenanz) [reply src/IO/ImageBase/ReadAnImage/Code.py:1]: Make a new PR, which adds the missing headers and modifies the corresponding .rst to skip including the licenses.

Comment (kian-weimer) [reply src/IO/ImageBase/ReadAnImage/Code.py:1]: Thank you, will do.

**Review by dzenanz — APPROVED** [review]

**Pull request merged**